README.md

```diff
@@ -55,18 +78,41 @@ chmod +x drive_linux
 This will output a url you need to visit and sign in, which will return an auth code to enter in the console.
 After you've entered the auto code, the credentials are stored in *gdrive_temp/.gd/credentials.json*.
 
+#### Login to OpenShift
 
-### Deployment Using the OpenShift Applier
+Login to OpenShift
 
-The project can be deployed using the [openshift-applier](https://github.com/redhat-cop/openshift-applier).
+```
+oc login <openshift url>
+```
 
-#### Prerequisites
+#### Deployment
 
-The following prerequisites must be satisfied:
+Utilize the following steps to deploy the project
 
-1. [Ansible](https://www.ansible.com/)
-2. OpenShift Command Line Interface (CLI)
-3. OpenShift environment
+1. Clone the repository
+
+    ```
+    git clone https://github.com/redhat-cop/ninja-board
+    ```
+
+2. Change into the project directory and utilize Ansible Galaxy to retrieve required dependencies
+
+    ```
+    ansible-galaxy install -r requirements.yml --roles-path=galaxy
+    ```
+
+3. Ensure the variables have been updated in the _ninja-board-deployment_ params file
+
+4. Execute the _openshift-applier_
+
+    ```
+    ansible-playbook -i .applier/inventory galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml -e="@.openshift/params/ninja-board-deployment" -e exclude_tags=ldap-rbac,ldap,v2
+    ```
+
+Once complete, all of the v1 resources should be available in OpenShift
+
+### OpenShift v2 Deployment
```

Comment:
I would change it to "v2 OpenShift Deployment" to avoid a "potential" confusion that this can only be deployed on OCP v2